### PR TITLE
provider/google: Cache http health checks.

### PR DESCRIPTION
**Pull request by duftler**
provider/google: Add support for configuring AutoHealering during create/clone.

### CODE REVIEW

**Comment by duftler** [comment]
@danielpeach please review.

File: app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js [hunk 1]
```diff
@@ -263,6 +269,29 @@ module.exports = angular.module('spinnaker.serverGroup.configure.gce.configurati
...
+      if (command.credentials === null) {
+        return result;
+      }
+
+      filteredData.httpHealthChecks = _(command.backingData.httpHealthChecks[0].results)
+        .filter({provider: 'gce', account: command.credentials})
+        .pluck('name')
+        .valueOf();
+
+      if (_.has(command, 'autoHealingPolicy.healthCheck') && !_(filteredData.httpHealthChecks).contains(command.autoHealingPolicy.healthCheck)) {
```
Comment (danielpeach) [inline app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js]: I think `contains` checks for equality only by reference.

**Comment by danielpeach** [comment]
LGTM after comment

File: app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js (hunk 1 above)
Comment (duftler) [inline app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js]: `healthCheck` is a string, so this should be fine.

**Pull request merged**